scripts/read-config.js

```diff
@@ -0,0 +1,39 @@
+import path from "path";
+import fs from "fs";
+import yaml from "js-yaml";
+
+const readConfig = () => {
+  if (process.env.NODE_ENV === "production") {
+    if (!process.env.CONFIG) {
```

Comment:
Let's refactor this into `readProductionConfig` and `readDevelopmentConfig` methods.

```
if (process.env.NODE_ENV === "production")
  readProductionConfig()
else
  readDevelopmentConfig()
end
```

Reply:
Thanks for pointing this out! Yea, that nested `if()` statement was a bit confusing to me as well.

Reply:
https://github.com/c-hive/dotdev/pull/138/commits/649b38d965aa3b8183721c14e5e9652ab67305eb